### PR TITLE
leo-editor: init at 5.1

### PR DESCRIPTION
**Pull request by leonardoce**

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -11845,6 +11845,8 @@ let
     inherit (gnome) libglade;
   };
 
+  leo_editor = callPackage ../applications/editors/leo-editor { };
```
Comment (lethalman) [inline pkgs/top-level/all-packages.nix]: You can name it `leo-editor` if you want.

Comment (leonardoce) [inline pkgs/top-level/all-packages.nix]: I didn't know that I could use a dash; I will rename it.
Thank you.

**Comment by leonardoce** [comment]
I've changed the name. Thanks.

**Comment by lethalman** [comment]
Travis passed, thanks.

**Pull request merged**